Update adding-organizations-to-your-enterprise-account.md

highfi13g opened the pull request:
The existing verbiage creates an absolute when noting "You cannot transfer existing organizations into an enterprise account on your end."  This is incorrect as Sales Operations has a process that they can use to assist with transfers as long as the organization is verified to be owned by the same customer. The updated tidbit provides more clarity and accuracy. Albeit i will take any constructive criticism on my update. 😸 ✌🏽 

cc:/ @stacycarter @github/support-squad-access 

<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to a pull request) and what changes you've made, we can triage your pull request to the best possible team for review.

See our [CONTRIBUTING.md](/main/CONTRIBUTING.md) for information how to contribute.

For changes to content in [site policy](https://github.com/github/docs/tree/main/content/github/site-policy), see the [CONTRIBUTING guide in the site-policy repo](https://github.com/github/site-policy/blob/main/CONTRIBUTING.md).

We cannot accept changes to our translated content right now. See the [contributing.md](/main/CONTRIBUTING.md#earth_asia-translations) for more information.

Thanks again!
-->

### Why:

<!-- 
- If there's an existing issue for your change, please link to it.
- If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->
**Closes [issue link]**

### What's being changed:

<!-- Share artifacts of the changes, be they code snippets, GIFs or screenshots; whatever shares the most context. -->

The below verbiage on [this page](https://docs.github.com/en/github/setting-up-and-managing-your-enterprise/adding-organizations-to-your-enterprise-account#:~:text=In%20the%20top%2Dright%20corner,a%20name%20for%20your%20organization)

```
To add an organization to your enterprise account, you must create the organization from within the enterprise account settings. You cannot transfer existing organizations into an enterprise account.
```

Is being updated to reflect the options available for those that want to transfer existing organizations to their Enterprise Account. Some companies may forget to transfer an organization or end up with new ones through acquisitions. 

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

highfi13g committed:
Update adding-organizations-to-your-enterprise-account.md

The existing verbiage creates an absolute when noting "You cannot transfer existing organizations into an enterprise account on your end."  This is incorrect as Sales Operations has a process that they can use to assist with transfers as long as the organization is verified to be owned by the same customer. The updated tidbit provides more clarity and accuracy.

janiceilene commented:
Thanks so much for opening a PR @highfi13g 🙌 I'll get this triaged for review ⚡

highfi13g committed:
Update adding-organizations-to-your-enterprise-account.md

As it will end up directly with a Sales representative anyway I am updating the verbiage i originally submitted.

hubwriter commented on content/github/setting-up-and-managing-your-enterprise/adding-organizations-to-your-enterprise-account.md:
```diff
@@ -11,7 +11,7 @@ versions:
 Enterprise owners can create new organizations within an enterprise account's settings.
 
-To add an organization to your enterprise account, you must create the organization from within the enterprise account settings. You cannot transfer existing organizations into an enterprise account.
+To add an organization to your enterprise account, you must create the organization from within the enterprise account settings. You cannot transfer existing organizations into an enterprise account on your end. If you need assistance transferring an existing organization please contact your GitHub Sales account representative.
```
For clarity, I think we should avoid using "on your end". I think we could just cut the existing second sentence and just mention contacting GitHub. How about:

```suggestion
To add an organization to your enterprise account, you must create the organization from within the enterprise account settings. If you want to transfer an existing organization to your enterprise account, contact your {% data variables.product.prodname_dotcom %} sales account representative.
```

I wonder whether everyone reading this will know who their sales account rep is. Would it be better to tell them to contact Support, as we do for many other things? 

i.e. 
> contact [{% data variables.product.prodname_dotcom %} Support](https://support.github.com/contact).

hubwriter committed:
Update content/github/setting-up-and-managing-your-enterprise/adding-organizations-to-your-enterprise-account.md

hubwriter reviewed (approved):
I've altered the wording slightly and committed the change, just so's we can get this merged.

hubwriter committed:
Merge branch 'main' into highfi13g-patch-1

hubwriter commented:
@highfi13g - Thanks for raising this PR. 👍  
I'm going to go ahead and merge this.
